microceph/ceph/operations.go

```diff
@@ -128,17 +124,29 @@ func (o *CheckNonOsdSvcEnoughOps) Run(name string) error {
 		"mgr": 0,
 		"mds": 0,
 	}
+	total_mon_services := 0
 	for _, service := range services {
-		// do not count the service on this node
+		// do not count the services on this node (these are the services remaining if this node is down)
 		if service.Location != name {
 			remains[service.Service]++
 		}
+
+		// count the total number of mon services, so we can calculate the minimum number of mon services required for quorum
+		if service.Service == "mon" {
+			total_mon_services += 1
+		}
 	}
 
+	// a majority of ceph-mon services must remain active to retain quorum
+	minMon := total_mon_services / 2 + 1
```

Comment:
May be this calculation should be lower bounded to 3? 

As recommended by official [docs](https://docs.ceph.com/en/mimic/rados/operations/add-or-rm-mons/#adding-monitors):

> We recommend at least 3 monitors for a production cluster

Reply:
This is about the minimum required to retain quorum, so the minimum theoretically is 1 (for a single node deployment). For a 2 or 3 node cluster, the minimum for quorum is 2. :)

Reply:
I should read the comment carefully 👀